src/components/ButtonPanel.js

```diff
@@ -14,22 +14,30 @@ const orangeButtons = [
   '÷', 'x', '-', '+',
 ];
 
-const ButtonPanel = () => (
+const keyList = [
+  'ed4b', 'fd9c', '62b6', '47c7', 'ad30',
+];
+
+const ButtonPanel = ({ clickHandler }) => (
```

Comment:
**REQUIRED**
* Implements `handleClick` method that returns `clickHandler` method from the props
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png)).
```javascript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
}
```
* It passes `handleClick` as `clickHandler` props to each `Button` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png))